recipes/iss/recipe.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import starfish
+from starfish.types import Axes
+
+
+def process_fov(field_num: int, experiment_str: str):
```

Comment:
two small tweaks I would suggest:

1. experiment path before field num.
2. rename `experiment_str` to `experiment_path`.